### PR TITLE
Frontend validation on uploading not supported file formats

### LINKED ISSUE #1193: No frontend validation on uploading not supported file formats
## Issue 
On uploading unsupported file formats, Magento tries to upload the asset, but server side declines the file. Need to have storefront validation.

## Steps 

- Open Media Gallery and click Upload images button
- Select files that are not supported 
Screen Shot 2020-04-13 at 1 59 36 PM

- Confirm selection and wait for Magento to return the backend validation:
Screen Shot 2020-04-07 at 11 16 59 AM

 ## ✅ Expected behavior 

- Consider option to disable not supported file formats from selection. 
- If user dragged the not supported files to upload, perform frontend validation
- If possible upload files with supported format and show success message  and block uploading not supported formats and  show error message with file types that are not supported. If above not possible , than block uploading all files and  show error message

## Design

Image-uploaded-messages

Basically, bulk the success messages into one message, but call out each failed file:

`Cannot upload “filename-of-the-image.mov”. This file format is not supported.`

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1193: No frontend validation on uploading not supported file formats
2. ...

### Manual testing scenarios (*)
- Consider option to disable not supported file formats from selection. 
- If user dragged the not supported files to upload, perform frontend validation
- If possible upload files with supported format and show success message  and block uploading not supported formats and  show error message with file types that are not supported. If above not possible , than block uploading all files and  show error message

## Design

Image-uploaded-messages

Basically, bulk the success messages into one message, but call out each failed file:

`Cannot upload “filename-of-the-image.mov”. This file format is not supported.`

### CODE REVIEW

File: MediaGalleryUi/view/adminhtml/web/js/image-uploader.js [hunk 1]
```diff
@@ -115,18 +115,19 @@ define([
          */
         showErrorMessage: function (data) {
             data.files.each(function (file) {
-                this.mediaGridMessages().add('error', $.mage.__('Cannot upload <b>' + file.name + '</b>. This file is not supported'));
+                this.mediaGridMessages().add(
+                    'error',
+                    $.mage.__('Cannot upload <b>' + file.name + '</b>. This file is not supported')
```
Comment (coderimus) [inline MediaGalleryUi/view/adminhtml/web/js/image-uploader.js]: Let's change the `This file is not supported` to `The file format is not supported` :)

**Review by coderimus — CHANGES_REQUESTED** [review]

File: MediaGalleryUi/view/adminhtml/web/js/image-uploader.js (hunk 1 above)
Comment (Nazar65) [reply MediaGalleryUi/view/adminhtml/web/js/image-uploader.js]: good catch :)

**Review by coderimus — APPROVED** [review]
@Nazar65 thank you for the work well done 👍

**Comment by coderimus** [comment]
The continious-integration/travis-ci-pr has status pending, but, in fact, it passed.
image

**Comment by chalov-anton** [comment]
:heavy_check_mark: QA Passed
file_upload_errors
